File: 1464-maximum-product-of-two-elements-in-an-array/1464-maximum-product-of-two-elements-in-an-array.java
class Solution {
    public int maxProduct(int[] nums) {
        int n = nums.length;
        int maxprod = Integer.MIN_VALUE;

        for (int i = 0; i < n - 1; i++) {
            for (int j = i + 1; j < n; j++) {
                int prod = (nums[i] - 1) * (nums[j] - 1);
                maxprod = Math.max(maxprod, prod);
            }
        }

        return maxprod;
    }
}

File: 1464-maximum-product-of-two-elements-in-an-array/1464-maximum-product-of-two-elements-in-an-array.js
var maxProduct = function(nums) {
    let maxProd = -Infinity;

    for (let i = 0; i < nums.length - 1; i++) {
        for (let j = i + 1; j < nums.length; j++) {
            let prod = (nums[i] - 1) * (nums[j] - 1);
            maxProd = Math.max(maxProd, prod);
        }
    }

    return maxProd;
};